### PR TITLE
Genius Yield: Add value locked in DEX Orders to the TVL Dashboard #8429

### PR DESCRIPTION
**Pull request by 4TT1L4**
Related to:
 - #8429

This PR is to add the TVL locked in the Genius Yield DEX.

The current adapter implementation collects only the value locked in staking. This must be extended with support for the value locked in the DEX.

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/genius-yield exports TVL: 
        
 
 
    cardano-staking           0
    cardano                   0
    staking                   0
    
    total                    0

File: projects/genius-yield/index.js [hunk 1]
```diff
@@ -1,17 +1,56 @@
...
+const stakingAddr = 'addr1w8r99sv75y9tqfdzkzyqdqhedgnef47w4x7y0qnyts8pznq87e4wh'
+const dexPaymentCredential = 'edfff663d37fc5f9753bc4222e0da2bfe08aa48db0837d2c329adeb3'
 
 module.exports = {
   timetravel: false,
   cardano: {
-    staking: sumTokensExport({ owner, tokens: [gens, nmkr, ntx, emp, gensx]}),
-    tvl: () => ({})
+    staking: sumTokensExport({ stakingAddr, tokens: [GENS, NMKR, NTX, EMP, GENSX]}),
+    tvl: sumTokensExport({ dexPaymentCredential, tokens: [AGIX, BOOK, CGV, CHRY, cNETA, COPI, DJED, EMP, ENCS, FACT, FLAC, GENS, GENSX, GXLA, HOSKY, HUNT, IAG, iBTC, iETH, INDY, iUSD, JELLY, JPG, LENFI, LIFI, LQ, MELD, MILK, MIN, NEWM, NINJAZ, NMKR, NTX, OPT, OPTIM, RJV, SHEN, SNEK, SOC, SUNDAE, VYFI, WMT, WRT]})
```
Comment (4TT1L4) [inline projects/genius-yield/index.js:54]: ADA is still missing.

There is also ADA locked into the DEX. Not only Cardano Native tokens, but also ADA.

Comment (4TT1L4) [reply projects/genius-yield/index.js]: Added 'lovelace' in d4c82ba0fa2ef67b5b89ab8558db975a6355dbdd to include ADA as well in the TVL.

**Comment by llamatester** [comment]
The adapter at projects/genius-yield exports TVL: 
        
 
 
    cardano-staking           0
    cardano                   0
    staking                   0
    
    total                    0

**Comment by llamatester** [comment]
The adapter at projects/genius-yield exports TVL: 
        
 
 
    cardano-staking           0
    cardano                   0
    staking                   0
    
    total                    0

**Comment by llamatester** [comment]
The adapter at projects/genius-yield exports TVL: 
        
 
 
    cardano                   30.96 k
    cardano-staking           7.17 M
    staking                   7.17 M
    
    total                    30.96 k

**Comment by llamatester** [comment]
The adapter at projects/genius-yield exports TVL: 
        
 
 
    cardano                   63.18 k
    cardano-staking           7.20 M
    staking                   7.20 M
    
    total                    63.18 k

**Comment by llamatester** [comment]
The adapter at projects/genius-yield exports TVL: 
        
 
 
    cardano                   63.19 k
    cardano-staking           7.22 M
    staking                   7.22 M
    
    total                    63.19 k

File: projects/genius-yield/index.js [hunk 2]
```diff
@@ -1,17 +1,56 @@
...
+const staking = 'addr1w8r99sv75y9tqfdzkzyqdqhedgnef47w4x7y0qnyts8pznq87e4wh'
+const orders = 'addr_vkh1ahllvc7n0lzljafmcs3zurdzhlsg4fydkzph6tpjnt0tx0asedu'
 
 module.exports = {
   timetravel: false,
   cardano: {
-    staking: sumTokensExport({ owner, tokens: [gens, nmkr, ntx, emp, gensx]}),
-    tvl: () => ({})
+    staking: sumTokensExport({ owner: staking, tokens: [GENS, NMKR, NTX, EMP, GENSX]}),
+    tvl: sumTokensExport({ owner: orders, tokens: ['lovelace', AGIX, BOOK, CGV, CHRY, cNETA, COPI, DJED, EMP, ENCS, FACT, FLAC, GENS, GENSX, GXLA, HOSKY, HUNT, IAG, iBTC, iETH, INDY, iUSD, JELLY, JPG, LENFI, LIFI, LQ, MELD, MILK, MIN, NEWM, NINJAZ, NMKR, NTX, OPT, OPTIM, RJV, SHEN, SNEK, SOC, SUNDAE, VYFI, WMT, WRT]})
```
Comment (g1nt0ki) [inline projects/genius-yield/index.js:54]: Hi @4TT1L4 the pr is mostly good, but I think specifying only the owner field of enough for tvl

Comment (4TT1L4) [reply projects/genius-yield/index.js:54]: Thanks for the hint. This helps a lot to keep the adapter easy to maintain. So the tokens don't need to be added every time we list a new pair. Great! Thank you.

**Comment by llamatester** [comment]
The adapter at projects/genius-yield exports TVL: 
        
 
 
    cardano                   65.62 k
    cardano-staking           7.42 M
    staking                   7.42 M
    
    total                    65.62 k

File: projects/genius-yield/index.js (hunk 2 above)
Comment (4TT1L4) [reply projects/genius-yield/index.js]: @g1nt0ki 
I have addressed your suggestion in the following commit: ff1c163c1fe18a46a9b44204450bd29f1e5ccc78

**Comment by g1nt0ki** [comment]
@4TT1L4 thanks for the PR

**Pull request merged**